app/src/main/kotlin/com/wire/android/ui/home/conversations/ConversationTopAppBar.kt

```diff
@@ -250,7 +250,7 @@ fun PreviewConversationScreenTopAppBarLongTitle() {
             conversationName = UIText.DynamicString(
                 "This is some very very very very very very very very very very long conversation title"
             ),
-            conversationDetailsData = ConversationDetailsData.Group(QualifiedID("", "")),
+            conversationDetailsData = ConversationDetailsData.Group(null, QualifiedID("", "")),
```

Comment:
can we add named params? thought it could be easier to understand what we are passing as `null`